frontend/src/components/QuestionCard.vue

```diff
@@ -23,7 +22,7 @@ defineProps<Props>();
     corner="top-left"
     border
     shadow
-    padding="medium"
+    :padding="Object('medium')"
```

Comment:
without Object it throws bunch of errors in the console. I don't understand the reasons really